docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -20,6 +20,41 @@ if you want to have products data exported to Elasticsearch after `build-demo` t
 
 ### Application
 - *(low priority)* if you want to test behavior of cart with no listable product in it, implement functional test as it is in [#852](https://github.com/shopsys/shopsys/pull/852)
+- copy data fixtures into your project namespace `ShopBundle\DataFixtures` from `FrameworkBundle\DataFixtures` of [shopsys/framework] repository ([#854](https://github.com/shopsys/shopsys/pull/854))
```

Comment:
That is not true, unfortunately. 
Imagine a situation where I have a project on Beta6 (Data fixtures are part of Framework). I update dependencies and start upgrading and get to this instruction.
I look into my vendor/shopsys/framework for data and they're not there. 
I look at Github to shopsys framework repository and they're not there either.

They had to copy them from project-base. Same applies for all following instructions.